academy/xl-run-prod/1-software.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Prepare the software to run
+order: 1
+description: Build and deploy
+tag: deep-dive
+---
+
+# Prepare the software to run
+
+To get to production, the first order of business is to build the binary that the nodes will run. If you used Ignite CLI, then you already did this under the hood with the `ignite chain serve` command.
+
+## Target platforms
+
+Because you are going to run the nodes on different machines, it is possible for them to have different operating systems and CPU types. You need to account for that when building. In particular, the computer on which you build may be entirely different from the one on which you eventually run the binary. In the jargon, you need to specify the target platform(s).
```

Comment:
```suggestion
Because you are going to run the nodes on different machines, it is possible for them to have different operating systems and CPU types. You need to account for that when building the binary. In particular, the computer on which you build may be entirely different from the one on which you eventually run the binary. In the jargon, you need to specify the target platform(s).
```